Configuration/PasswordSequenceConfiguration.cs

```diff
@@ -13,6 +16,23 @@ public PasswordSequenceConfiguration()
 
             Sequence = new List<SequenceItem>();
         }
+        
+        /* this constructor is for creating an empty configuration if reading
+         * the config file fails somehow; the argument is unused but I don't
+         * know a better way to do it
+         */
+        public PasswordSequenceConfiguration(bool throwaway)
+        {
+            XmlNamespace = new XmlSerializerNamespaces();
+            XmlNamespace.Add(string.Empty, "http://quasivirtuel.com/PasswordSequenceConfiguration.xsd");
+
+            Sequence = new List<SequenceItem>();

```

Comment:
I didn't know the syntax for that, thank you. If you can't tell, I'm not very experienced with C#. My coding life has revolved around C for over a decade, and even my C++ is quite rusty. I've updated with this fix.